src/sagemaker/jumpstart/constants.py

```diff
@@ -149,3 +151,10 @@ class VariableScope(str, Enum):
 
     CONTAINER = "container"
     ALGORITHM = "algorithm"
+
+
+class JumpStartTag(str, Enum):
+    """Enum class for tags to apply to JumpStart models."""
```

Comment:
Is that the tag key? If yes, can you update the docstring to make it clear please?